### ISSUE TITLE
Improve Islands map

### ISSUE THREAD

**Issue by deferred-impact** [deferred-impact/battledudes-issue-tracker#67]
Ice made Islands *almost* a map i would actually vote for unironically... but it's still not quite there.

Having thought about it for some time, i realized the issue. Even with ice now replacing water and making the iced area of the map actually tolerable, there's... **still no reason to go there**. The islands are very close to each other, and are arranged in a line, making it not just trivial, but in fact optimal, to mostly ignore the ice except for the few channels between said islands.

Here is the proposed solution.

![image](https://user-images.githubusercontent.com/87503574/189500480-84c1ecb1-84a4-4f9e-a40b-2c6d4128936d.png)

By arranging 4 islands in the corners of the map, we ensure that the entire map is meaningful. Why would you go over the ice now? Well, why wouldn't you? It is the shortest path between diagonal islands. At the same time, people who dislike ice can still mostly avoid it, by only traversing the short channels inbetween orthogonally adjacent islands.

**Comment by snipergamingop** [comment]
Yes please the map itself is basically worthless to play on. If this is implemented then I think it would be one of the most voted map in the game

**Comment by Gregory-Y** [comment]
Good idea definitely think this should be added. Though I would also add something in the middle (not sure what) because it shouldn't be COMPLETELY empty.

**Comment by deferred-impact** [comment]
I don't see any problem with an empty middle. In fact, i would argue that a large skating ring surrounded by islands is preferable to 5 islands with tiny canals between them.

**Comment by DisposableKatyusha** [comment]
Maybe add a bit of little land strips the middle for an attack base strategy

**Comment by deferred-impact** [comment]
The purpose of this change is to force people to visit the ice
Adding land in the middle defeats the purpose
